docs/tools/tools_intro.md

```diff
@@ -11,7 +11,7 @@ The three Mbed OS development tools are Mbed Studio, Mbed CLI and the Mbed Onlin
 
 We developed Mbed OS using the Mbed CLI tool, which is a Python program that coordinates builds and fetches all the dependencies of an Mbed OS application. As this runs on your local development machine, you also need compilers and other build tools installed.
 
-`os.mbed.com` provides the tools, libraries and programs that work with Mbed OS, so you can also use Mbed Studio or the Mbed Online Compiler for building Mbed OS examples and programs. Beginner developers or those who are not comfortable with the command-line may prefer Mbed Studio or the Online Compiler. Furthermore, you can use the exporters to third party development tools that were part of the Arm Mbed OS 2 ecosystem.
+`os.mbed.com` provides the tools, libraries and programs that work with Mbed OS, so you can also use Mbed Studio or the Mbed Online Compiler for building Mbed OS examples and programs. Beginner developers or those who are not comfortable with the command-line may prefer Mbed Studio or the Online Compiler. 
```

Comment:
```suggestion
`os.mbed.com` provides the tools, libraries and programs that work with Mbed OS, so you can also use Mbed Studio or the Mbed Online Compiler for building Mbed OS examples and programs. Beginner developers or those who are not comfortable with the command-line may prefer Mbed Studio or the Online Compiler. Furthermore, you can use the exporters to third party development tools.
```
Maybe add a link to the exporter page at the end of the paragraph.

Reply:
I'm not sure the exporter information fits here. There is a paragraph about exporting at the end of the page.